src/vis.ts

```diff
@@ -0,0 +1,38 @@
+import * as THREE from 'three';
+import * as plane from './plane';
+import {OrbitControls} from 'three/examples/jsm/controls/OrbitControls';
+
+export class Vis {
+  renderer: THREE.WebGLRenderer;
+  scene: THREE.Scene;
+  camera: THREE.PerspectiveCamera;
+  aircraft: THREE.Group;
+  camControls: OrbitControls;
+
+  constructor() {
+    const width = window.innerWidth,
```

Comment:
This visualization is being added to an element within the window, but its dimensions are being set to the dimensions of the window, limiting its flexibility as a class. I'd recommend setting the dimensions as part of the operation where you add it to the DOM.